src/kernel/capinv.c

```diff
@@ -930,6 +930,8 @@ cap_introspect(struct captbl *ct, capid_t capid, u32_t op, unsigned long *retval
 		return thd_introspect(((struct cap_thd *)ch)->t, op, retval);
 	case CAP_TCAP:
 		return tcap_introspect(((struct cap_tcap *)ch)->tcap, op, retval);
+	default:
+		return -EINVAL;
 	}
 	return -EINVAL;
```

Comment:
What's the point having `default: return -EINVAL;` directly before `return -EINVAL`?

Reply:
Added this due to new gcc compilation warnings about enum cases not being handled. Removed the second `return -EINVAL` outside of the switch statement.